host/greatfet/commands/greatfet_dac.py

```diff
@@ -37,7 +37,7 @@ def main():
 
     # 0 - 1024
     dac_value = 1024
```

Comment:
This value is hard coded but should be taken as an argument

Reply:
see #134